docarray/typing/tensor/tensorflow_tensor.py

```diff
@@ -298,3 +298,6 @@ def unwrap(self) -> tf.Tensor:
 
     def __len__(self) -> int:
         return len(self.tensor)
+
+    def __eq__(self: T, other: T) -> np.ndarray:
+        return tf.equal(self.tensor, other.tensor).numpy()
```

Comment:
```suggestion

    def __eq__(self: T, other: T):
        return self.tensor == other.tensor
```

Reply:
Changed!

Reply:
@samsja after changing to your suggestion, the error occured which tells `TensorFlowTensor` class didn't have tensor attribute. Is it necessary to change it become tensor attribute?